src/assets/storage.ts

```diff
@@ -0,0 +1,33 @@
+import { bnToBn } from "@polkadot/util"
+
+import { AssetBalanceType } from "./types"
+
+import { chainQuery, txPallets } from "../constants"
+import { BalanceType, query } from "../blockchain"
+
+/**
+ * @name getTotalAssetBalance
+ * @summary           The holdings of a specific account for a specific asset.
+ * @param assetId     The ID of the asset.
+ * @param AccountId   Public address of the account to get balances.
+ * @returns           The holdings/balance information of the account : balance, isFrozen: boolean, reason, extra
+ */
+export const getTotalAssetBalance = async (assetId: number, AccountId: string): Promise<AssetBalanceType> => {
```

Comment:
The return can be `null`, the return type has to be: `Promise<AssetBalanceType | null>`